### PR TITLE
Add MFA docs

### PR DESCRIPTION
**Pull request by glena**
Add MFA docs

## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/docs/pulumi-cloud/accounts.md [hunk 1]
```diff
@@ -71,6 +71,22 @@ For users who signed up with email, you may change your password by:
 1. Navigate to **Settings** > **Account security**
 1. Reset your password. Your password must be at least ten characters long.
 
+## Setting up second factor authentication
+
+For users who signed up with email, you can enable second factor authentication using one time passwords:
+
+1. Use the organization selector to choose your individual account.
+1. Navigate to **Settings** > **General**
+1. On the right column, click on the **Enroll** button under **Two factor authentication**
```
Comment (meagancojocar) [inline themes/default/content/docs/pulumi-cloud/accounts.md:80]:
```suggestion
3. Scroll to the MFA section
4. Press Enroll
```

File: themes/default/static/images/docs/pulumi-cloud/mfa/mfa-enroll.png
Comment (meagancojocar) [inline themes/default/static/images/docs/pulumi-cloud/mfa/mfa-enroll.png:1]: We should probably re-name this section to MFA. I understand today we just have 2 factors of auth but we need a consistent brand across all and MFA is more common.

File: themes/default/content/docs/pulumi-cloud/accounts.md [hunk 2]
```diff
@@ -71,6 +71,22 @@ For users who signed up with email, you may change your password by:
...
+![MFA Enroll](/images/docs/pulumi-cloud/mfa/mfa-enroll.png)
+1. Download a TOTP authenticator app, for example, you can use Google Authenticator (available for <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2">Android</a> and <a href="https://apps.apple.com/us/app/google-authenticator/id388497605">IPhone</a>)
```
Comment (meagancojocar) [inline themes/default/content/docs/pulumi-cloud/accounts.md:82]:
```suggestion
1. Download a TOTP authenticator app, for example, you can use Google Authenticator (available for <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2">Android</a> and <a href="https://apps.apple.com/us/app/google-authenticator/id388497605">iOS</a>)
```

**Review by meagancojocar — APPROVED** [review]
Some minor edits!

File: themes/default/static/images/docs/pulumi-cloud/mfa/mfa-enroll.png
Comment (glena) [reply themes/default/static/images/docs/pulumi-cloud/mfa/mfa-enroll.png:1]: I just created a PR, I will update the screenshots in this PR in a minute

image

https://github.com/pulumi/pulumi-service/pull/17695

**Pull request merged**